### DISCUSSION TITLE
Labels and Icons Not Appearing

### DISCUSSION THREAD

**Discussion by dpl-friday** [Budibase/budibase#15121] [Help]
Hi, I'm running into an issue where certain labels and icons in Budibase aren't showing up as expected. Couple examples:

1. In a table component with "Delete rows" turned on, the delete icon doesn't appear in preview mode after selecting the row
1. When hovering over a component to edit, I only see a couple of text descriptions, but not the full set of options or icons that should normal appear

Has anyone else experienced this? I am using Chrome on Windows as my browser and we use a self-hosted set up.

**Comment by mikesealey** [comment]
Hey @duc-bhcoe this sounds like it could be a bug - please could you submit a new issue, and follow the template? Extra points if you can use https://www.jam.dev to provide us with a screen capture. Jam is a particularly powerful tool, as it captures your screen and also the developer console and network tabs, which may hold some answers.

**Comment by mikesealey** [comment]
Hey @duc-bhcoe - just a heads up, we still haven't recieved the password for the encrypted app you sent us. I've got some capacity to look into it this afternoon.

**Reply by dpl-friday** [reply]
Sorry for the delay @mikesealey,
Actually, the issue occured in the same app that @hkim-jadehealth and I working on.
I believed @hkim-jadehealth already sent you the exported app with the password?

Two issues we has opened:
- https://github.com/Budibase/budibase/issues/15157
- https://github.com/Budibase/budibase/issues/15132
